### DISCUSSION TITLE
Map Interface

### DISCUSSION THREAD

**Discussion by Nisgrak** [directus/directus#3133] [Feature Requests]
I'm testing the new v9 (very good job guys, amazing!) but I missing the map interface, has been deprecated?

**Comment by benhaynes** [comment]
Not deprecated, just hasn't been added in yet. We're hoping to get it in before 9.0.0 is officially released, but would love any PRs! 😉  The main thing to consider is that we don't want to have any paid libraries or required API tokens to 3rd party services within the Core platform. I forget which OpenMap we were using in v8, but we'd likely want to keep the same one in v9 (but with updated styling to match the new version).

**Reply by Nisgrak** [reply]
Nice! Didn't find any estimated date, have a estimation? Thanks!

**Reply by benhaynes** [reply]
We can only provide delivery dates for _sponsored_ work, all open-source work and contributions are triaged and prioritized internally. Dates change, and so we've decided not to give estimates any more for non-contract updates.

**Comment by rijkvanzanten** [comment]
The map interface has since been added!
